inference-engine/tests/functional/inference_engine/ngraph_reader/deformable_convolution_opset8_tests.cpp

```diff
@@ -0,0 +1,214 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <string>
+#include "ngraph_reader_tests.hpp"
+
+TEST_F(NGraphReaderTests, ReadDeformableConvolution8Network) {
+    std::string model = R"V0G0N(
+<net name="deformable_convolution" version="10">
+	<layers>
+		<layer id="0" name="in1" type="Parameter" version="opset8">
+			<data shape="1,256,200,336" element_type="f32"/>
+			<output>
+				<port id="0" precision="FP32">
+					<dim>1</dim>
+					<dim>256</dim>
+					<dim>200</dim>
+					<dim>336</dim>
+				</port>
+			</output>
+		</layer>
+		<layer id="1" name="in2" type="Parameter" version="opset8">
+			<data shape="1,18,100,168" element_type="f32"/>
+			<output>
+				<port id="0" precision="FP32">
+					<dim>1</dim>
+					<dim>18</dim>
+					<dim>100</dim>
+					<dim>168</dim>
+				</port>
+			</output>
+		</layer>
+		<layer id="2" name="in3" type="Parameter" version="opset8">
+			<data shape="256,256,3,3" element_type="f32"/>
+			<output>
+				<port id="0" precision="FP32">
+					<dim>256</dim>
+					<dim>256</dim>
+					<dim>3</dim>
+					<dim>3</dim>
+				</port>
+			</output>
+		</layer>
+	    <layer id="3" name="1090" type="DeformableConvolution" version="opset8">
+            <data deformable_group="1" dilations="1,1" group="1" pads_begin="1,1" pads_end="1,1" strides="2,2"/>
```

Comment:
@ilyachur do we need such test? I suppose we agreed to have only visitor api tests for operations

Reply:
@itikhono Yes, we don't need to add new `ngraph_reader` tests. visitor tests should cover this functionality

Reply:
the tests have been deleted